.travis.yml

```diff
@@ -15,33 +15,38 @@ php:
 sudo: false
 
 env:
+  matrix:
+    - UNISH_NO_TIMEOUTS=y UNISH_DRUPAL_MAJOR_VERSION=8 UNISH_DB_URL=mysql://root:@127.0.0.1 PHPUNIT_ARGS=tests/configTest.php TEST_CHILDREN="drush-ops/config-extra"
```

Comment:
What exactly does this line do? Most of the params are repeated below. Is this effectively a way to get TEST_CHILDREN added to each line below? If so, could we not do same for UNISH_TIMEOUTS? I have tried to use matrix before but always been stymied by https://github.com/travis-ci/travis-ci/issues/1519